### ISSUE TITLE
how show the predicted reselts in the image ?urgent

### ISSUE THREAD

**Issue by hfahad** [brianhuang1019/FCN-pytorch#7]
hello can you please tell me how we show the image with mask of the predicted data 
i mean how you show these results ?

**Comment by pochih** [comment]
You mean the results?
You can save the output of pre-trained model.
